Library/SPIClient.m

```diff
@@ -1178,7 +1178,11 @@ - (BOOL)send:(SPIMessage *)message {
  Set the acquirer code of your bank, please contact mx51's Integration Engineers for acquirer code.
```

Comment:
Sorry Doni, my fault, I forgot to add the comment commit in. Are you able to update and add in the comments for the `setAcquirerCode` and `setTenantCode` like here, please? https://github.com/mx51/spi-client-js/blob/feature/v2.9.0/src/Spi.js#L187-L197